client/ayon_core/plugins/publish/extract_otio_review.py

```diff
@@ -491,3 +492,19 @@ def _get_ffmpeg_output(self):
             out_frame_start = self.used_frames[-1]
 
         return output_path, out_frame_start
+
+    def _get_unique_file_prefix(self, instance):
+        """Creates unique human readable file prefix to differentiate.
+
+        Multiple instances might share same temp folder, this will provide
+        unique prefix for intermediate file for burnins.
+        """
+        folder_path = instance.data["folderPath"]
+        folder_name = folder_path.split("/")[-1]
+        folder_path = folder_path.replace("/", "_").lstrip("_")
+
+        file_prefix = f"{folder_path}_{folder_name}."
+        self.log.debug(f"file_prefix::{file_prefix}")
+
+        return file_prefix
```

Comment:
Does this _really_ enforce uniqueness? Is there no way one folder comes up more than once but with multiple media?

Can't we just use some "temp" file name but with a prefix that we want, like whatever Python `tempfile` would generate as path. If you provide a prefix it'll still attach some actual temp identifier after it?